tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -22,13 +22,11 @@ There is a walkthrough through the process below in the [how do I use the test p
 2. Install test-proxy
 
 ```powershell
-> dotnet tool install azure.sdk.tools.testproxy --global --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json
+> dotnet tool install azure.sdk.tools.testproxy --global --prerelease --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json
 ```
 
 This feed is available in [the public azure-sdk project.](https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk)
 
-Note that given there are only dev versions available, please add `--version <selectedVersion>`. You will see a failure that **gives you the latest version of the package** if you do not provide this additional argument.
```

Comment:
Don't we generally want most usages to pick a particular version instead of floating to the latest?

Reply:
Depends on the scenario.  Ad-hoc on a dev machine I would recommend latest.  In a pipeline where we want repeatable builds, then we should pin.  But we typically use the docker image instead of `dotnet tool` in pipelines.

My goal for this PR was to simplify the ad-hoc dev-machine scenario.

Reply:
Given that this readme is more for `adhoc` use, I'm ok with this change. 🤷‍♂️

Reply:
I agree for local dev scenarios folks generally just prefer the latest and I'm good with these changes and we can always make the exact version/command available in CI logs if folks need to debug a particular issue that only appears in CI.